ci-operator/step-registry/stackrox/operator-e2e/stackrox-operator-e2e-ref.yaml

```diff
@@ -23,6 +23,8 @@ ref:
     default: ''
   - name: COLLECTION_METHOD
     default: core_bpf
+  - name: WORKER_NODE_COUNT
+    default: '4'
```

Comment:
This will not affect the node count as the cluster is provisioned elsewhere. https://github.com/openshift/release/blob/4d62a37c08894c9c6337d81aafd4c93f438f0a9e/ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-15-lp-interop.yaml#L47
```suggestion
```